src/createStore.js

```diff
@@ -1,8 +1,11 @@
 import { createStore as originalCreateStore } from 'redux'
 
 export function createStore (reducer) {
-  const extension = window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
-  return originalCreateStore(reducer, extension)
+  let enhancer
+  if (process.env.NODE_ENV !== 'production' && window.__REDUX_DEVTOOLS_EXTENSION__) {
+    enhancer = window.__REDUX_DEVTOOLS_EXTENSION__()
```

Comment:
I agree for real apps.

I kept it here so that the readers can try out the DevTools right on `gh-pages` without having to compile and run the app themselves. I also kept it in MobX version.